apps/desktop/src/components/charts/HistoricalAPMChart.tsx

```
@@ -0,0 +1,293 @@
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  // Format period for display on x-axis
+  const formatPeriodDisplay = (period: string, scale: string): string => {
+    switch (scale) {
+      case 'daily':
+        // Convert 2025-01-26 to Jan 26
+        const date = new Date(period);
+        return date.toLocaleDateString('en-US', { month: 'short', day: 'numeric' });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)